rj_gameplay/stp/utils/constants.py

```diff
@@ -46,6 +49,32 @@ class BallConstants:
 
 class EvaluationConstants:
     SLOW_THRESHOLD = 1
+    SETTLE_BALL_SPEED_THRESHOLD = 1.0
```

Comment:
I know you didn't pick the name, but can you rename EvaluationConstants to something more relevant? SLOW_THRESHOLD is unused, so you can delete it, and SETTLE_BALL_SPEED_THRESHOLD is part of settle skill. 

Also, INVALID_COST is actually unnecessary since role assignment was changed, so you can delete it/references to it.